website/cue/reference/components/sinks/websocket_server.cue

```diff
@@ -119,5 +119,35 @@ components: sinks: websocket_server: {
 				it requires Basic auth with defined username and password.
 				"""
 		}
+		message_buffering: {
+			title: "Message buffering"
+			body: """
+				The `message_buffering` configuration option can be used to enable this feature. It can
+				be used to define a number of events to be buffered, which would enable replay for clients that
+				may want to continue from the last message after disconnection. To provide clients with the
+				message ID, `message_buffering.message_id_path` needs to be defined, which is used
+				to encode the ID inside outgoing messages. The buffer is backed by a ring buffer, so
+				the oldest messages will be lost when the size limit is reached.
+
+				Once clients have the ID, on future connections that the ID can be sent in the
+				`last_received` query parameter and all buffered messages since that message are
+				sent to the client immediately on connection. If the message can't be found, the entire
+				buffer is replayed.
```

Comment:
```suggestion
				After clients have the ID, they can send it in the `last_received` query parameter on future connections.
				All buffered messages since that ID are then sent to the client immediately upon connection.
				If the message can't be found, the entire buffer is replayed.
```